src/app/functions/server/getBreaches.ts

```diff
@@ -49,50 +44,3 @@ export async function getBreaches() {
 
```

Comment:
I can't add a comment to line 37, but I think we'll want to set `breach.LogoPath = null;` there — otherwise it'll point to the HIBP URL, instead of our cached favicon.

Reply:
line taken out